haiku-apps/medo/patches/medo-1.0.0~beta1.2.patchset

```diff
@@ -0,0 +1,22 @@
+From 820f05d3bcf4418716b3c0876bcb37d55523fefc Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Thu, 18 Feb 2021 17:47:26 +0100
+Subject: x86 build fix
+
+
+diff --git a/Jamfile b/Jamfile
+index 0ef01c3..dbad1d1 100644
+--- a/Jamfile
++++ b/Jamfile
+@@ -10,7 +10,7 @@ else
+ 	C++FLAGS += -std=c++2a -g -O0 -gno-column-info ;
+ }
+ C++FLAGS += -I. -IYarra -I/boot/system/develop/headers/private/interface ;
+-C++FLAGS += -I/boot/system/develop/headers/freetype2 ;
++C++FLAGS += -I/boot/system/develop/headers/freetype2 -I/boot/system/develop/headers/x86/freetype2;
```

Comment:
This should fix the "build":
```diff
- +C++FLAGS += -I/boot/system/develop/headers/freetype2 -I/boot/system/develop/headers/x86/freetype2;
+ +C++FLAGS += -I/boot/system/develop/headers/freetype2 -I/boot/system/develop/headers/x86/freetype2 ;
```

Reply:
@pulkomandy you added the **-maxv** last time for this, the build completed here OK (on R1Beta2), would this still be needed?

Reply:
I added it because the compiler was complaining otherwise. If it doesn't complain anymore I suspect the problem is fixed?